.circleci/config.yml

```diff
@@ -691,7 +691,7 @@ jobs:
             TEST_FILES=$(echo "$TEST_FILES" | circleci tests split --split-by=timings)
             TEST_FILES=$(echo "$TEST_FILES" | sed 's|^test/||')
             MATCH_PATH="./test/{$(echo "$TEST_FILES" | paste -sd "," -)}"
-            forge test --match-path "$MATCH_PATH"
+            forge test --match-path "$MATCH_PATH" -j $(nproc)
```

Comment:
I think forge does this by default if not specified

Reply:
ok will remove